src/objects.rs

```diff
@@ -657,6 +657,25 @@ pub struct Coord {
     pub lat: f64,
 }
 
+// Mean earth radius from International Union of Geodesy Geophysics
+const EARTH_RADIUS: f64 = 6_371.000_8;
+
+impl Coord {
+    /// Calculate the orthodromic distance in kilometers
```

Comment:
Better to do it the navitia way: pure metric system, i.e. meter, meter/second for speed, etc.

Reply:
We generally call this formula haversine distance https://docs.rs/geo/0.8.2/geo/algorithm/haversine_distance/trait.HaversineDistance.html

Reply:
I don't see your point. This is the same formula with different radius.
You want me to add rust geo ?